atmo/atmo_test.go

```diff
@@ -0,0 +1,176 @@
+package atmo
+
+import (
+	"bytes"
+	"net/http"
+	"testing"
+
+	"github.com/suborbital/atmo/atmo/options"
+	"github.com/suborbital/vektor/vtest"
+)
+
+//I think the easiest way would be to make a single Atmo method that does everything Start() does but instead looks something like func (a *Atmo) testStart() (*vk.Server, err), and then you can call that method from your atmo_test.go tests
```

Comment:
These comments are good, but I think best left on the GH issue rather than in the source code